core-ui/src/components/Lambdas/LambdaDetails/Tabs/ResourceManagement/ResourceManagement/shared.js

```diff
@@ -64,40 +61,44 @@ function prepareSchema() {
       .transform((val, originalVal) => {
         return originalVal === '' ? -1 : val; // -1 so that instead of throwing errors about NaN it will pass validation here, but fail on min(0) with nicer error message
       })
-      .positive(errorMessages.MIN_REPLICAS_POSITIVE)
-      .integer(errorMessages.MIN_REPLICAS_POSITIVE)
-      .test('matchMinReplicas', errorMessages.MIN_REPLICAS_TOO_HIGH, function(
-        arg,
-      ) {
-        return arg <= this.parent.maxReplicas;
-      }),
+      .positive(t('functions.create-view.errors.min-replicas-positive'))
+      .integer(t('functions.create-view.errors.min-replicas-positive'))
+      .test(
+        'matchMinReplicas',
+        t('functions.create-view.errors.min-replicas-too-high'),
+        function(arg) {
+          return arg <= this.parent.maxReplicas;
+        },
+      ),
     [inputNames.replicas.max]: yup
       .number()
       .transform((val, originalVal) => {
         return originalVal === '' ? -1 : val; // -1 so that instead of throwing errors about NaN it will pass validation here, but fail on min(0) with nicer error message
       })
-      .positive(errorMessages.MAX_REPLICAS_POSITIVE)
-      .integer(errorMessages.MAX_REPLICAS_POSITIVE)
-      .test('matchMaxReplicas', errorMessages.MAX_REPLICAS_TOO_LOW, function(
-        arg,
-      ) {
-        return arg >= this.parent.minReplicas;
-      }),
+      .positive(t('functions.create-view.errors.max-replicas-positive'))
```

Comment:
it's not translated

Reply:
orrr.. aparently if both `max`/`min` errors are shown, one of them is translated properly.. so there is a different issue, not with just the translation itself

Reply:
same thing for `memory-default`, `memory-default` etc